sdk/cosmos/azure_data_cosmos/src/clients/container_client.rs

```diff
@@ -38,11 +40,13 @@ impl ContainerClient {
             .feed(ResourceType::Containers)
             .item(container_id);
         let items_link = link.feed(ResourceType::Items);
+        let request_handler = RequestHandler::new(pipeline.clone());
```

Comment:
Let's just move the `pipeline` into the `RequestHandler` rather than cloning

Reply:
As discussed, for simplicity we can remove the dependency on request_handler for now. We can add it back later if needed.

Therefore this change is now no longer applies.